reactiveweb/src/get-promise-state.ts

```diff
@@ -206,6 +206,33 @@ export type GetPromiseStateInput<Value> =
  * NOTE: This `getPromiseState` is not a replacement for [WarpDrive](https://docs.warp-drive.io/)'s [getRequestState](https://www.npmjs.com/package/@warp-drive/ember#getrequeststate)
  *       namely, the `getPromiseState` in this library (reactiveweb) does not support futures, cancellation, or anything else specific to warp-drive.
  *
+
+| . | reactiveweb | @warpdrive/ember |
+| - | ----------- | ---------------- |
+| use in module state[^module-state] | ✅ | ✅ |
+| use in a getter[^cached-getter] | ✅ | ✅ |
+| usable in template | ✅ | ✅  |
+| immediate has resolved value for resolved promise | ✅  | ✅  |
+| invokes a passed function automatically | ✅ | ❌ |
+| avoids addons using private APIs | ✅ | ❌[^private-apis] |
+| test waiter integration | ✅ | ✅ |
+| no dependencies[^no-dependencies] | ✅[^ember-resources] | ❌ |
```

Comment:
having a dependency on the framework that you should already have installed is not bad, also there is no dependency - `getPromiseState` actually comes from core, its just re-exported for Ember.